rega-genotype/src/main/java/rega/genotype/utils/Settings.java

```diff
@@ -187,10 +187,13 @@ public static Settings getInstance(ServletContext context) {
         if (configFile == null) {
             String osName = System.getProperty("os.name");
             osName = osName.toLowerCase();
+            configFile = "etc/development-config.xml";
```

Comment:
I suppose this was a change you made to have it work locally? If yes, please remove it from the commit. Otherwise it can stay.